app/views/mailer/mfa_required_soon_announcement.html.erb

```diff
@@ -0,0 +1,118 @@
+<% @title = "Enable multi-factor authentication (MFA) on your RubyGems account" %>
+<% @sub_title = "👋 Hi #{@user.handle}" %>
+
+<!-- Body -->
+<table width="100%" border="0" cellspacing="0" cellpadding="0" bgcolor="#ffffff">
+  <tr>
+    <td class="content-spacing" style="font-size:0pt; line-height:0pt; text-align:left" width="20"></td>
+    <td>
+      <table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="35" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+
+      <table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="40" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+
+      <div class="h3-1-center" style="color:#1e1e1e; font-family:Georgia, serif; min-width:auto !important; font-size:20px; line-height:26px;">
+        <p>
+          Recently, we've <a href="https://blog.rubygems.org/2022/06/13/making-packages-more-secure.html" target="_blank">announced</a> our security-focused ambitions to the community.
+        </p>
+        <br/>
+
+        <p>
+          Next week, on August 15, 2022, we will begin requiring maintainers like yourself, who are owners of packages with more than 180 million downloads, to have multi-factor authentication (MFA) enabled.
+        </p>
+        <br/>
+
+        <p>
+          If you don't have MFA enabled by this date, a number of operations will be restricted.
+          These restrictions include accessing profile pages on the web, signing in on the command line, and performing <a href="https://guides.rubygems.org/mfa-requirement-opt-in/#privileged-operations" target="_blank">privileged actions</a> (ie. push, yank, add/remove owners).
+        </p>
+        <br/>
+
+        <% if @user.mfa_disabled? %>
+          <p> 
+            To avoid any disruptions with your RubyGems account, <b>please enable multi-factor authentication 🙏.</b>
+          </p>
+          <br/>
+        <% elsif @user.mfa_ui_only? %>
+          <p>
+            We're notifying you of this policy because you have MFA enabled only for the UI.
+            <b>Please upgrade your RubyGems account to a stronger MFA level 🙏.</b>
```

Comment:
nit: add what levels they should change it to (Ui and api or ui and gem signin)

Reply:
I must've misinterpreted your earlier [comment](https://github.com/rubygems/rubygems.org/pull/3166#discussion_r938933448), as we outlined what they should change it to, but I then removed it. Now, I'm thinking what you meant was to reword it?

Reply:
@jenshenny & I touched base IRL. We aligned it makes sense to remove the MFA level recommendation for folks who currently have MFA disabled, since they only have the two choices (`ui and api` and `ui and gem signin`).

However, for the messaging to folks who _have_ MFA enabled, but need to upgrade from `ui_only`, it may be unclear what "strong MFA" means. So it'd be more helpful to explicitly state that either `ui and api` or `ui and gem signin` are both accepted and recommended levels.